src/tests/ftest/util/command_utils.py

```diff
@@ -107,7 +107,11 @@ def __str__(self):
         if isinstance(self._default, bool) and self.value:
             return self._str_format
         elif not isinstance(self._default, bool) and self.value is not None:
-            if isinstance(self.value, (list, tuple)):
+            if isinstance(self.value, dict):
+                return " ".join(
+                    [self._str_format.format("{} \"{}\"".format(key, value))
+                     for key, value in self.value.items()])
```

Comment:
(pylint-no-member) Instance of 'list' has no 'items' member
(pylint-no-member) Instance of 'str' has no 'items' member
(pylint-no-member) Instance of 'bool' has no 'items' member